src/components/film/year-minutes/year-minutes.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
 import {Label} from 'src/components/common/label/label.js';
 
-const YearMinutes = ({year, minutes}) => {
-  return (
+export const YearMinutes = ({year, minutes}) => (
     <div className='year-min'>
       <Label content={year.slice(0,4)} labelWhite labelBold />
```

Comment:
year will be `"unkn"` for default props

Reply:
Corrected - props {year, minutes} are made required

Reply:
And what if they will be missing? Could you be sure that they will be always in place?